### DISCUSSION TITLE
Essentials Messages-Edit Problems

### DISCUSSION THREAD

**Discussion by emanuelsonf** [EssentialsX/Essentials#5798] [Q&A]
Hello everyone! I'm trying to edit essentials messages, so i uploaded the file message_it.properties and i modified some messages like ban, tempban, mute etc... but when i try to edit tpa and msg messages, the result in the game isn't shown, it always is how the default message! Please help me!

**Comment by ImDarkLaw** [comment]
For support inquiries that might require some back-and-forth, it's best to swing by our [Discord](https://discord.gg/PHpuzZS) server.
